v2/dns.go

```diff
@@ -10,11 +10,11 @@ type DNSResource struct {
 }
 
 type (
-	// SplitDnsRequest is a map from domain names to a list of nameservers.
-	SplitDnsRequest map[string][]string
+	// SplitDNSRequest is a map from domain names to a list of nameservers.
+	SplitDNSRequest map[string][]string
```

Comment:
As this is an exported symbol, this is a breaking change for users of the library. We have a brief note in the readme about "use at your own risk", but perhaps we should be more explicit that users should expect breaking changes to the V2 client API until we tag v2.0.0 for the repo?

Reply:
I've added a note about breaking changes to the README and godoc.